index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+the <a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+Implementers might find these other specifications through other standards
+bodies or communities that utilize this specification.
+              </li>
+            </ol>
+          </li>
+          <li>
+Determine the specific cryptographic verification algorithm
+(<var>verifyProof</var>) required to verify the securing mechanism:
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the Verifiable Credential Data Integrity
+Specification [[?VC-DATA-INTEGRITY]] defines the
+<a data-cite="VC-DATA-INTEGRITY#verify-proof">process for identifying
+a specific cryptography suite type</a> used for cryptographic verification by
+utilizing the `proof`.`type` property. A list of registered proof types can be
+found in the
+<a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+                <ol class="algorithm">
+                  <li>
+Set <var>securedDocument</var> to the result of a JSON parsing function that
+takes <var>inputBytes</var> as an input parameter, parses the value as JSON,
+and returns the result as an object. If JSON parsing fails,
+<var>securingMechanismVerified</var>.<var>status</var> is set to `false`, and a
+<a href="#PARSING_ERROR">PARSING_ERROR</a> is appended to
+<var>securingMechanismVerified</var>.<var>errors</var>.
+                  </li>
+                  <li>
+If <var>securedDocument</var> was successfully set in the previous step, set
+<var>securingMechanismVerified</var>.<var>status</var> to the result of passing
+<var>securedDocument</var>, along with any relevant function-specific options,
+to the <var>verifyProof</var> function. For example, the
+<var>expectedProofPurpose</var> option is set to `assertionMethod` when
+the conforming document is expected to be a <a>verifiable credential</a>,
+and if the <a>conforming document</a> is expected to be a
+<a>verifiable presentation</a>, the <var>expectedProofPurpose</var> option
+is set to `authentication` and the <var>domain</var> and <var>challenge</var>
+are also provided as options.
+                  </li>
+                </ol>
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+identification of a specific cryptography suite and cryptographic verification
+process.
+<div class="issue" title="Document verification process in vc-jose-cose">
+The process for securing mechanism verification is not clearly documented in
+the [[?VC-JOSE-COSE]] specification yet, but once it is, a link to that section
+of the specification will be included in this text.
+</div>
```

Comment:
Again, please remove the external references to securing specifications.  This is a layering violation.  It's up to the securing specifications to define how to verify that the payload is secured.

Reply:
I raised a PR to at least give us a thing to point too:

https://github.com/w3c/vc-jose-cose/pull/190


but I agree this issue marker should be reworked.

Reply:
Please provide a concrete re-working of the issue marker.